pyproject.toml

```diff
@@ -53,3 +54,7 @@ wkalamine = "kalamine.cli_msklc:cli"
 
 [tool.isort]
 profile = "black"
+
+[[tool.mypy.overrides]]
+module = ["progress.bar"]
+ignore_missing_imports = true
```

Comment:
why do we need this here ?
and what does the double [[ ]] mean ?

Reply:
`[[]]` means array, needed for this setting

and this is needed to avoid mypy to check `progress` code it seems. Otherwise I get an error